src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -1238,6 +1239,10 @@ class DocBaseViewer extends BaseViewer {
      * @return {void}
      */
     toggleThumbnails() {
+        if (!this.thumbnailsSidebarEl) {
+            return;
```

Comment:
This will prevent `resize()` from occurring if the thumbnails are missing. IIRC, this guarantees that Preview scales properly inside of the parent container.